hypothesis/via
# Remove all JSTOR code as LMS now provides a URL

For:

 * https://github.com/hypothesis/private-issues/issues/110

I've asked JSTOR and they are happy for these short lived URLs to be exposed publicly.

### Testing notes

 * Get the latest dev data (after https://github.com/hypothesis/devdata/pull/52)
 * All the usual `make dev` magic to stand everything up
 * Make sure your dev tools are open with the cache disabled
 * Visit: https://hypothesis.instructure.com/courses/125/assignments/2968
 * You should see a document, nothing externally should be any different
 * If you inspect the proxied URL you will see the S3 temp URL passed from LMS
